frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedItems/ChartSettingOrderedItems.tsx

```diff
@@ -46,6 +49,7 @@ const SortableColumn = SortableElement(function SortableColumn<
   return (
     <ColumnItem
       title={getItemName(item)}
+      extra={getItemExtra ? getItemExtra(item) : null}
```

Comment:
According to `SortableColumnProps`, `getItemExtra` cannot be `undefined`

```suggestion
      extra={getItemExtra(item)}
```

Although I see TS complaining about missing `getItemExtra` in `frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx`.